frontend/package.json

```diff
@@ -6,6 +6,7 @@
     "start": "ng serve --aot",
     "build": "ng build --aot",
     "test": "ng test",
+    "test:ci": "ng test --watch=false --browsers=ChromeHeadlessCustom",
```

Comment:
What do you think about turning on headless browser as a default for local tests as well?

Reply:
Yeah, good for me! I've changed it.